backend/openssl.cnf

```diff
@@ -0,0 +1,362 @@
+#
```

Comment:
is it normal to have that in the public repo?
This is a production conf no?

Reply:
I agree with you on the principle that we should provide a configuration alternative for each of the service provided by the docker-compose file, but I think this is out of scope for this PR.